src/tests/JIT/opt/Devirtualization/box2.cs

```diff
@@ -17,7 +17,10 @@ static async Task<int> Main()
             // extra context argument.
             await new ValueTask<string>(Task.Delay(1).ContinueWith(_ => default(string))).ConfigureAwait(false);
         }
+    }
 
-        return 100;
+    public static void TestEntryPoint()
```

Comment:
`Fact`? Also, `Task.Run(TestTask)` should presumably be `Task.Run(TestTask).Wait()`.

Reply:
thanks - fixed